VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
+---
+title: Excel performance: Performance and limit improvements
+description: This article discusses performance improvements in Microsoft Excel 2016 and Microsoft Excel 2010. This article is one of three companion articles about techniques that you can use to improve performance in Excel as you design and create worksheets.
```

Comment:
Let's use a more concise, action-oriented summary. Something like "Find out about performance improvements in Excel." Generally phrases like "This article discusses..." dilute SEO. Not sure that we need to mention that there are companion articles in the search description. Also, we no longer include "Microsoft" with product names.